src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -358,8 +358,13 @@ public boolean accept(RawPacket pkt)
         SimulcastController simulcastController
             = ssrcToBitrateController.get(ssrc);
 
-        return simulcastController != null
-            && simulcastController.accept(pkt);
+        if (simulcastController == null) {
+            logger.warn("Dropping an RTP packet, because the SSRC has not " +
```

Comment:
@bgrozev i do worry a bit that this log might get too verbose.  i wonder if we should try and only log this every X packets or something.